machine/wnc_x86_64/INSTALL

```diff
@@ -0,0 +1,36 @@
+=============================================
+Installing ONIE on WNC x86_64
```

Comment:
Okay, after refer Accton's solution, I will modify it looks like machine/vic/vic_model, where vic is temp vendor name.